scripts/pre_commit_linter.py

```diff
@@ -1685,7 +1562,7 @@ def _check_codeowner_file(verbose_mode_enabled):
         python_utils.PRINT('Starting CODEOWNERS file check')
         python_utils.PRINT('----------------------------------------')
 
-    with _redirect_stdout(_TARGET_STDOUT):
+    with _redirect_stdout(sys.stdout):
```

Comment:
Why we made this change in this place not in other places in the same file?

Reply:
That was a mistake, will fix this.